src/Symfony/Component/Console/Input/StringInput.php

```diff
@@ -24,7 +24,9 @@
  */
 class StringInput extends ArgvInput
 {
-    public const REGEX_STRING = '([^\s\\\\]+?)';
+    // @deprecated constant replaced by REGEX_UNQUOTED_STRING, to be removed in Symfony 7.0
```

Comment:
let's remove this line, we'll add it back in 6.1 (or please allow me to push on your fork ;) )

Reply:
You should be able to edit the PR. Otherwise I can update in a few hours.

Reply:
Updated!